doc/en/user/source/installation/war.rst

```diff
@@ -3,9 +3,9 @@
 Web archive
 ===========
 
-GeoServer is packaged as a standalone servlet for use with existing application servers such as `Apache Tomcat <http://tomcat.apache.org/>`_ and `Jetty <http://eclipse.org/jetty/>`_.
+GeoServer is packaged as a standalone servlet for use with existing application servers such as `Apache Tomcat <https://tomcat.apache.org/>`_ and `Jetty <https://jetty.org/>`_.
 
-.. note:: GeoServer has been mostly tested using Tomcat, and so is the recommended application server. GeoServer requires a newer version of Tomcat (7.0.65 or later) that implements Servlet 3 and annotation processing. Other application servers have been known to work, but are not guaranteed.
+.. note:: GeoServer has been mostly tested using Tomcat, and so is the recommended application server. Current versions of GeoServer require Tomcat version 9.x. This version is recent enough to implement Servlet 3 and annotation processing, but has not yet made the change from ``java`` to ``jakarta``, which `GeoServer has also not yet made <https://github.com/geoserver/geoserver/wiki/Jakarta-EE>`_. Other application servers have been known to work, but are not guaranteed.
```

Comment:
actually it would be `javax`, 
```suggestion
.. note:: GeoServer has been mostly tested using Tomcat, and so is the recommended application server. Current versions of GeoServer require Tomcat version 9.x. This version supports Servlet API 3 and annotation processing that GeoServer implements, but has not made the change from Java EE to Jakarta EE, which `GeoServer has also not yet made, see:  <https://github.com/geoserver/geoserver/wiki/Jakarta-EE>`_. Other application servers have been known to work, but are not guaranteed.
```